debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -680,6 +749,17 @@ public static OversizeHandlingMode parse(String value, String defaultValue) {
             .withInvisibleRecommender()
             .withDescription("A comma-separated list of regular expressions that match the collection names for which changes are to be excluded");
 
+    public static final Field FILTERS_MATCH_MODE = Field.create("filters.match.mode")
+            .withDisplayName("Database and collection include/exclude match mode")
+            .withEnum(FiltersMatchMode.class, FiltersMatchMode.REGEX)
+            .withGroup(Field.createGroupEntry(Field.Group.FILTERS, 6))
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("The mode used by the aggregation pipeline to match events based on  included/excluded database and collection names"
```

Comment:
nit: Extra space:
> based on__included/excluded

Reply:
We should probably mention which properties this effects explicitly.

Reply:
These are not full docs but rather brief description used in connector config schema.

Reply:
Gotcha, I was getting confused between Debezium and Flink, where the latter uses such constructs to generate actual customer facing docs.